torch_geometric/nn/models/explainer.py

```diff
@@ -388,3 +388,52 @@ def visualize_subgraph(self, node_idx: Optional[int], edge_index: Tensor,
         nx.draw_networkx_labels(G, pos, **label_kwargs)
 
         return ax, G
+
+    def visualize_feature_importance(
```

Comment:
Lets add this to the new `Explainer` class in `torch_geometric.explain`, since we are going to depreciate this in the future.

Reply:
Sounds good to me! I'll update and push!